react-components/src/metadata/admin/AttributeEnumEditGrid.js

```diff
@@ -77,20 +67,20 @@ class AttributeEnumEditDialog extends Component {
             >
                 <DialogTitle id={dialogTitleID}>{getMessage("dialogTitleEditEnumValue")}</DialogTitle>
                 <DialogContent>
-                    <Field name={`${field}.value`}
-                           label={formatMessage(intl, "value")}
-                           id={build(dialogID, ids.ENUM_VALUE)}
-                           required={true}
-                           autoFocus
-                           margin="dense"
-                           component={FormTextField}
+                    <FastField name={`${field}.value`}
+                               label={formatMessage(intl, "value")}
+                               id={build(dialogID, ids.ENUM_VALUE)}
+                               required={true}
+                               autoFocus
+                               margin="dense"
+                               component={FormTextField}
                     />
                     <Field name={`${field}.is_default`}
                            label={formatMessage(intl, "enumDefaultLabel")}
                            id={build(dialogID, ids.ENUM_VALUE_DEFAULT)}
                            color="primary"
                            component={FormCheckbox}
-                           normalize={this.normalizeDefaultField}
+                           onClick={normalizeDefault(enumValues, replace)}
```

Comment:
Unfortunately an attribute enum `is_default` checkbox (here and in the enum listing grid) can't be a `FastField`, otherwise this enum values list passed to the `normalizeDefault` function can become stale, and the `normalizeDefault` function won't work properly (if a value becomes checked, this function needs to uncheck any other value with the FieldArray's `replace` function, since there can be only one default checked).